lib/tasks/green_lanes.rake

```diff
@@ -121,20 +121,27 @@ namespace :green_lanes do
     # load existing data
     theme = GreenLanes::Theme.where(section: '1', subsection: '3').first
     assessments = GreenLanes::CategoryAssessment.all.index_by do |assessment|
-      [assessment.measure_type_id, assessment.regulation_id]
+      [assessment.measure_type_id, assessment.regulation_id, assessment.regulation_role]
     end
-    types = %w[551 552 553 554 555 561 562 564 565 566 570 695]
+    types = %w[551 552 553 554 555 561 562 564 565 566 570 690 695 696]
 
     GreenLanes::CategoryAssessment.db.transaction do
       current_date = Time.zone.today
       actual_condition = Sequel.lit('validity_start_date <= ? AND ' \
                                       '(validity_end_date IS NULL OR validity_end_date > ?)')
 
+      # select active measure_id, regulation_id and regulation_role of given measure type to create category_assessment
```

Comment:
Sorry should've specified what I meant: a comment to explain what we're doing business-wise. Like what you said on the PR "[...] filter out inactive regulations"

Reply:
The new changes don't require a comment now. So please remove it 😊